docs/api-docs/analytics/bodl-for-storefronts.mdx

```diff
@@ -16,37 +19,100 @@ For scripts to successfully use BODL to access storefront data, the following mu
 | `channel_id` | string | The BigCommerce channel ID. Channel ID of the channel where checkout is taking place. | [Get all channels](/docs/rest-management/channels#get-all-channels) <br /> `data.id` |
-| `coupon_codes[]` | array of strings | Coupon codes applied to the cart. | [Get a cart](/docs/rest-management/carts/carts-single#get-a-cart) <br /> `coupons.code` |
-| `currency` | string | [ISO-4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) of the order. BigCommerce tracks currency for a cart and not for each line item within the cart. | [Get a cart](/docs/rest-management/carts/carts-single#get-a-cart) <br /> `currency.code` |
+| `currency` | string | [ISO-4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) of the order. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br />  `data.cart.currency.code` |
+| `id` | string | Unique ID for each event instance. | Unique event ID generated by BigCommerce. |
+| `line_items[]` | array of `line_item` objects | Items added to the cart. | See [line_item](#line_item-object-schema) object for event fields |
+| `product_value` | number | Item price before discounts and coupons. Product default price or, if applicable, the sale price configured in the admin panel. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `lineItems.physicalItems.listPrice` or <br /> `lineItems.digitalItems.listPrice`|
+
+### View cart
+
+When the shopper clicks a button to view a cart, the following fields appear in the BODL object for the item:
+
+| Event field | Type | Description | BigC data map |
+|:---|:---|:---|:---|
+| `cart_value` | number | Final value of cart after taxes, discounts, and coupons are applied. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `cartAmount` |
+| `channel_id` | string | The BigCommerce channel ID. Channel ID of the channel where checkout is taking place. | [Get all channels](/docs/rest-management/channels#get-all-channels) <br /> `data.id` |
+| `currency` | string | [ISO-4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) of the order. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br />  `data.cart.currency.code` |
+| `id` | string | Unique ID for each event instance. | Unique event ID generated by BigCommerce. |
+| `line_items[]` | array of `line_item` objects | All items in the cart. | See [line_item](#line_item-object-schema) object for event fields |
+
+### Remove product from cart
+
+When the shopper clicks a button to remove a product from the cart, the following fields appear in the BODL object for the item:
+
+| Event field | Type | Description | BigC data map |
+|:---|:---|:---|:---|
+| `cart_value` | number | Final value of cart after taxes, discounts, and coupons are applied. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `cartAmount` |
```

Comment:
```suggestion
| `product_value` | number | Final value of cart after taxes, discounts, and coupons are applied. | [Get a cart](/docs/rest-storefront/carts#get-a-cart) <br /> `cartAmount` |
```

Reply:
@bc-sanjayjoshi @bc-rmalyavc Even though the [google doc](https://docs.google.com/document/d/13qlYa9gtuVcjs2iBZ-jWP74W1GPRQw9J496t7QYbbJg/edit#) says `product_value`, I get `cart_value` 
![remove product from cart](https://user-images.githubusercontent.com/96258747/227644666-ff5b64a9-6b17-41e9-a0df-a6d102e5c3db.png)

Reply:
@bc-rmalyavc Maybe a bug? product_value would be for 1 product and cart_value for the entire cart. Only time they would be the same is if the cart had 1 product.

Reply:
Resolving conversation, problem fixed after I updated my stencil theme